### PR TITLE
contract: Translations fix

### PR DESCRIPTION
**Pull request by hhgabelgaard**

### CODE REVIEW

File: contract/models/contract.py [hunk 1]
```diff
@@ -614,14 +614,14 @@ def _add_contract_origin(self, invoices):
                 move.message_post(
                     body=(
                         _(
-                            (
-                                "%(msg)s by contract <a href=# data-oe-model=contract.contract"
-                                " data-oe-id=%(contract_id)d>%(contract)s</a>."
-                            ),
-                            msg=move._creation_message(),
```
Comment (pedrobaeza) [inline contract/models/contract.py:621]: AFAIK, this one is correct.

File: contract/models/contract_line.py [hunk 2]
```diff
@@ -889,9 +889,9 @@ def cancel(self):
             raise ValidationError(_("Cancel not allowed for this line"))
         for contract in self.mapped("contract_id"):
             lines = self.filtered(lambda l, c=contract: l.contract_id == c)
-            msg = _(
-                "Contract line canceled: %s",
-                "<br/>- ".join(
+            msg = (
+                _("Contract line canceled: %s")
+                % "<br/>- ".join(
```
Comment (pedrobaeza) [inline contract/models/contract_line.py]: And this syntax is also valid and don't included in the translation. Have you tried?

File: contract/models/contract.py (hunk 1 above)
Comment (hhgabelgaard) [reply contract/models/contract.py:621]: I don't think so. It's not in .pot file

Comment (pedrobaeza) [reply contract/models/contract.py:621]: Uhm, but you didn't update POT files with this PR to check if it's working.

File: contract/models/contract_line.py (hunk 2 above)
Comment (hhgabelgaard) [reply contract/models/contract_line.py]: Your right, This is included in the .pot file
I'll remove my changes here

File: contract/models/contract.py (hunk 1 above)
Comment (hhgabelgaard) [reply contract/models/contract.py:621]: Yes, the change is included in a regenerated .pot file.
Do we update .pot/.po file by PR's?
Or only by the GH action and weblate?

Comment (pedrobaeza) [reply contract/models/contract.py:621]: Well, I wanted to make sure is working, but on merge, they will be regenerated.

**Review by pedrobaeza — APPROVED** [review]

**Pull request closed without merging**